ember-core/src/main/scala/org/http4s/ember/core/Parser.scala

```diff
@@ -127,7 +127,7 @@ private[ember] object Parser {
         isChunked =
           headers
             .get(org.http4s.headers.`Transfer-Encoding`)
-            .exists(_.value.toList.contains(TransferCoding.chunked))
+            .exists(_.values.exists(_ === TransferCoding.chunked))
```

Comment:
Wait, what? Did that convert to a `String` and then to a `Seq[Char]`?

Reply:
I think so I didn't get the typing working, but I have no idea how it worked.